wpcc.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
 
   s.test_files = Dir['spec/**/*']
 
-  s.add_dependency 'dough-ruby'
+  s.add_dependency 'dough-ruby', '~> 5.25'
```

Comment:
Should the readme be updated or how do we let everyone know/when to update the version? tagging @johnplayer1982 @davidtrussler @Tr4pSt3R